app/models/version.rb

```diff
@@ -309,6 +309,11 @@ def recalculate_metadata!
     nil
   end
 
+  def link_uri(name, linkset = nil)
+    metadata[name.to_s].presence || linkset.try(name).presence ||
+      (name.to_s == "documentation_uri" ? documentation_path : nil)
```

Comment:
It would be nice to remove this and read better.

`linkset#documentation_uri` has access to `rubygem#name`, but does not have access to `version#number`. Ideas?

```ruby
  def documentation_path
    "http://www.rubydoc.info/gems/#{rubygem.name}/#{number}"
  end
```